internal/testrunner/runners/system/tester.go

```diff
@@ -1646,7 +1646,7 @@ func (r *tester) validateTestScenario(ctx context.Context, result *testrunner.Re
 	}
 
 	if r.fieldValidationMethod == mappingsMethod {
-		logger.Warn("Validation based on mappings enabled (technical preview)")
+		logger.Debug("Performing validation based on mappings")
```

Comment:
I thought that maybe this message could help to identify if the validation based on mappings has been executed or not. WDYT ?